content/en/blog/2019/dns-cert/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Istio DNS certificate management
+description: Provision and manage DNS certificates in Istio.
+publishdate: 2019-10-29
+attribution: Lei Tang (Google)
+keywords: [security, kubernetes, certificates, DNS]
+target_release: 1.4
+---
+
+The DNS certificates used by Istio control plane components (e.g., Galley, Sidecar Injector)
+are by default managed by Citadel, which is a large component, maintains its own private
+signing key, and also acts as a CA.
+
+In Istio 1.4, we introduce a new feature to securely provision and manage DNS certificates
+signed by Kubernetes CA in Istio, which has the following advantages.
+
+* This feature removes the dependency on Citadel for DNS certificates and is
+more lightweight than Citadel.
+
+* Unlike Citadel, this feature doesn't require maintaining a private signing key, which enhances security.
```

Comment:
```suggestion
* No longer requires you to maintain a private signing key, which enhances security.
```

Reply:
The current sentence is more clearer since "you" in the suggested text is unclear (Citadel maintains a private key, not you).